xml_inscriptions/transcribed/MA.Glouc.HCM.L.Tmp97.6.8-mdn.xml

```diff
@@ -144,8 +144,8 @@
                <lb n="3" break="no"/>grippa<supplied reason="omitted">e</supplied></name></persName>, Divi <expan><abbr>A<sic>gu</sic></abbr><ex>usti</ex></expan>
                <lb n="4"/>nepotis, uxor
                <lb n="5" break="no"/>is <persName><name type="cognomen" key="Germanicus">Germanici</name>
-               <lb n="6"/><name type="cognomen" key="Caesar">C<sic>ea</sic>sari</name></persName>, matris
-               <lb n="7"/><persName><name type="praenomen" key="Gaius"><expan><abbr>G</abbr><ex>ai</ex></expan></name> <name type="cognomen" key="Caesar"><expan><abbr>C<sic>ea</sic></abbr><ex>sa</ex><abbr>r</abbr><ex>is</ex></expan></name> <name type="cognomen" key="Augustus"><expan><abbr>Aug</abbr><ex>usti</ex></expan></name>
+               <lb n="6"/><name type="cognomen" key="Caesar">C<sic>ea</sic>sari<supplied reason="omitted">s</supplied></name></persName>, matris
```

Comment:
Shouldn't the markup here include that you're supplying from an alternate copy? Should have `<supplied reason="omitted" evidence="parallel">`

Reply:
Yes, I'll go back and fix it.